README.md

```diff
@@ -49,6 +49,14 @@ To install, *first modify the `values.yaml` file to add your license*:
 
 Follow the instructions in the Notes that follow the installation to find your Tyk login credentials.
 
+## Install Tyk Hybrid Gateways (This can be used either for Hybrid Gateways connected to Tyk Cloud or MDCB Hybrid Gateways)
+
+To install, *first modify `values_hybrid.yaml` file as follows:*
+1. Add your dashboard users organisation ID in `gateway.rpc.orgID` value
+2. Add your dashboard users API key in `gateway.rpc.apiKey` value 
```

Comment:
This should mention setting `gateway.rpc.connString` too, given that it's now variable.